### PR TITLE
Add new ccx and ccx_sensitive ACL configuration and new memory limit …

### PR DESCRIPTION
**Pull request by rimolive**
…for queries

### CODE REVIEW

File: kfdefs/base/trino/trino-config-secret.yaml [hunk 1]
```diff
@@ -77,6 +80,11 @@ stringData:
           "schema": "telemetry",
           "owner": true
         },
+        {
+          "group": "ccx",
+          "schema": "ccx|ccx_sensitive",
+          "owner": true
```
Comment (accorvin) [inline kfdefs/base/trino/trino-config-secret.yaml:86]: Is this right? We want everyone in the CCX group to be an owner of the schemas?

Comment (rimolive) [reply kfdefs/base/trino/trino-config-secret.yaml:86]: Ownership has a different meaning in this ACL configuration, but you made me review the configuration and it's not what we discussed in the sync call. Only ccx-admin would be able to create the tables whereas the others should only get read access. I made that change.

**Review by anishasthana — APPROVED** [review]

**Comment by anishasthana** [comment]
Manually merged as ci is broken

**Pull request merged**